### PR TITLE
Display services as services and not devices

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

Display services as services and not devices

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

image

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/integrations/ha-integration-card.ts [hunk 1]
```diff
@@ -183,7 +184,19 @@ export class HaIntegrationCard extends LitElement {
...
+                            "ui.panel.config.integrations.config_entry.services",
+                            "count",
+                            services.length
+                          )}</a
+                        >
+                      `
+                    : ""}
+                  ${(devices.length || services.length) && entities.length
                     ? this.hass.localize("ui.common.and")
                     : ""}
```
Comment (bramkragten) [inline src/panels/config/integrations/ha-integration-card.ts:201]: We should also render an `and` when there is both a service and a device.

Comment (ludeeus) [reply src/panels/config/integrations/ha-integration-card.ts:201]: I had that first, but if the integration has both you end up with:
"x services and x devices and x entities" which feels weird :shrug: 
Add a comma maybe?

Comment (bramkragten) [reply src/panels/config/integrations/ha-integration-card.ts:201]: yeah:

`2 devices, 1 service and x entities`

File: src/panels/config/integrations/ha-integration-card.ts [hunk 2]
```diff
@@ -184,6 +184,7 @@ export class HaIntegrationCard extends LitElement {
                         >
                       `
                     : ""}
+                  ${devices.length || services.length ? "," : ""}
```
Comment (bramkragten) [inline src/panels/config/integrations/ha-integration-card.ts:187]:
```suggestion
                  ${devices.length && services.length ? "," : ""}
```

Comment (ludeeus) [reply src/panels/config/integrations/ha-integration-card.ts:187]: :facepalm:

File: src/panels/config/integrations/ha-integration-card.ts [hunk 3]
```diff
@@ -184,6 +184,7 @@ export class HaIntegrationCard extends LitElement {
...
                   ${services.length
```
Comment (bramkragten) [inline src/panels/config/integrations/ha-integration-card.ts]:
```suggestion
                    : ""}${devices.length && services.length ? ", " : ""}${services.length
```

**Review by bramkragten — APPROVED** [review]

**Pull request merged**